splunk_add_on_ucc_framework/templates/globalConfig.json.init-template

```diff
@@ -49,9 +49,18 @@
                     ],
                     "title": "Accounts"
                 },
+                {%- if need_proxy %}
+                {
+                    "type": "loggingTab"
+                },
+                {
+                    "type": "proxyTab"
+                }
+                {%- else %}
                 {
                     "type": "loggingTab"
                 }
+                {%- endif %}
```

Comment:
Can we move `{%- if need_proxy %}` after `loggingTab`? We have `loggingTab` by default and it is being referenced in 2 places with this approach.

Reply:
Made the changes.